test/cases/runner-debug/runner.debug.spec.js

```diff
@@ -64,4 +65,40 @@ describe('CLI Fixture w/debug (stdOut) enabled', function() {
       ]);
     });
   });
-});
\ No newline at end of file
+
+});
+
+describe('CLI Fixture with output handlers', function() {
+  
+  it('should invoke the callback with the output', async function() {
+    const runner = new Runner();
+
+    await runner.runCommand(
+      `${fixturesPath}/cli-writes-to-stdout.js`,
+      null,
+      { async: true }
+    );
+
+    const output = runner.getStdOut();
+    const error = runner.getStdErr();
+
+    expect(output).to.be.equal('test');
+    expect(error).to.be.empty;
+  });
+
+  it('should invoke the callback with the error', async function() {
+    const runner = new Runner();
+
+    await runner.runCommand(
+      `${fixturesPath}/cli-writes-to-stderr.js`,
+      null,
+      { async: true, ignoreErrors: true }
```

Comment:
Added `ignoreErrors` so this wouldn't have to be wrapped in a silent try-catch block. See https://github.com/thescientist13/gallinago/pull/63/commits/e6fb36e3321aef26164d44feeb9a6ef2cacd497f

I think there's probably a bunch of other good reasons clients would want the option of not throwing when something gets written to stderr. I'm open to reverting this change though if you think I should open a separate issue / PR to discuss it.